site/en/docs/privacy-sandbox/user-agent/index.md

```diff
@@ -2,39 +2,39 @@
 layout: 'layouts/doc-post.njk'
 title: 'User-Agent reduction'
 subhead: >
-  Limit passively shared browser data to reduce the volume of sensitive information which leads to fingerprinting.
+  User-Agent reduction limits passively shared browser data to reduce the volume of sensitive information which leads to fingerprinting. The reduction is now complete.
 description: >
-  Limit passively shared browser data to reduce the volume of sensitive information which leads to fingerprinting.
+  User-Agent reduction limits passively shared browser data to reduce the volume of sensitive information which leads to fingerprinting. The reduction is now complete.
 date: 2021-11-09
-updated: 2023-08-01
+updated: 2023-09-23
 authors:
   - alexandrawhite
 ---
 
-## Implementation status
+<!-- ## Implementation status
+
+The following list indicates previous stages in the gradual User-Agent string reduction. The reduction is now complete and the origin and deprecation trials are closed.
```

Comment:
Is this meant to be hidden?

Reply:
I wanted to indicate that I think it should go

Reply:
removed status